yt/frontends/owls/fields.py

```diff
@@ -326,14 +326,12 @@ def _get_owls_ion_data_dir(self):
 
         # get test_data_dir from yt config (ytcgf)
         # ----------------------------------------------
-        tdir = ytcfg.get("yt", "test_data_dir")
+        data_dir = ytcfg.get("yt", "test_data_dir")
 
-        # set download destination to tdir or ./ if tdir isnt defined
+        # set download destination to cwd if test_data_dir is undefined
         # ----------------------------------------------
-        if tdir == "/does/not/exist":
```

Comment:
Comparing to the placeholder value (whether it's valid or not) is just wrong. It should have been `if not os.path.isdir(tdir)` from the very beginning. Could you change that in your version of the code?

Reply:
I was looking for the most conservative changes possible but you're absolutely right.

Reply:
(note that I'm not sure this won't cause an error on windows however)